bond/commands/groups_clear.py

```diff
@@ -0,0 +1,33 @@
+import bond.proto
+from bond.commands.base_command import BaseCommand
+from bond.database import BondDatabase
+
+
+class GroupsClearCommand(BaseCommand):
+    subcmd = "groups_clear"
+    help = "Remove all Bond's devices."
```

Comment:
Shouldn't it be group?
```suggestion
    help = "Remove all Bond's groups."
```

Reply:
It should! Thanks!